Project.toml

```diff
@@ -1,6 +1,6 @@
 name = "IntervalMatrices"
 uuid = "5c1f47dc-42dd-5697-8aaa-4d102d140ba9"
-version = "0.6.2"
+version = "0.6.3"
```

Comment:
Just to check: This PR did not bring any code changes, so the new version is unrelated, right?

Reply:
yep, this PR is only for docs changes (i added the logo and other minor changes). i'll make a new patch release ..

Reply:
.. because we need [this change](https://github.com/JuliaReach/IntervalMatrices.jl/pull/159/files) to be released